assets/js/base/components/totals/totals-item/index.js

```diff
@@ -16,23 +17,29 @@ const TotalsItem = ( { className, currency, label, value, description } ) => {
 			className={ classnames( 'wc-block-totals-table-item', className ) }
 		>
 			<span className="wc-block-totals-table-item__label">{ label }</span>
-			<FormattedMonetaryAmount
-				className="wc-block-totals-table-item__value"
-				currency={ currency }
-				displayType="text"
-				value={ value }
-			/>
-			<span className="wc-block-totals-table-item__description">
+			{ isNumber( value ) ? (
+				<FormattedMonetaryAmount
+					className="wc-block-totals-table-item__value"
+					currency={ currency }
+					displayType="text"
+					value={ value }
+				/>
+			) : (
+				<div className="wc-block-totals-table-item__value">
+					{ value }
+				</div>
+			) }
```

Comment:
Why this condition? Is it because the value might already be ready for display? If this is because `value` might be a react element, would `isValidElement` work here (from `@wordpress/element`) and then we can remove the `isNumber` and lodash dependency?

Reply:
Sometimes a "value" might be something non-numeric, such as the "calculate" link added in this PR. Happy to adjust if this is not appropriate.

Reply:
👍   I'm  wondering if it'd be more appropriate to check if the incoming value is a `validElement` and use that to determine whether to feed the value to FormattedMonetaryAmount or not.  The main benefit will be removing the need to have a dependency on lodash here (albeit I realize that lodash is pretty much a guaranteed dependency now anyways).

This isn't a blocker, we can leave as is being that there's not a significant reason to switch the check atm (the dependency switch has slight benefit).